apps/test/unit/facilitatorProgramRegistration.js

```diff
@@ -0,0 +1,235 @@
+import {assert} from '../util/configuredChai';
+const testUtils = require('./../util/testUtils');
+testUtils.setExternalGlobals();
+import React from 'react';
+import {mount} from 'enzyme';
+
+import FacilitatorProgramRegistration from '@cdo/apps/code-studio/pd/facilitator_program_registration/FacilitatorProgramRegistration';
+
+const DEFAULTS = {
+  apiEndpoint: "",
+  course: "",
+  options: {
```

Comment:
nope! Options are by design actually supplied by the server (from `Pd::FacilitatorProgramRegistration::OPTIONS`)